src/iop/filmicrgb.c

```diff
@@ -3582,7 +3582,7 @@ void gui_init(dt_iop_module_t *self)
   g->gui_sizes_inited = FALSE;
 
   // don't make the area square to safe some vertical space -- it's not interactive anyway
-  g->area = GTK_DRAWING_AREA(dtgtk_drawing_area_new_with_aspect_ratio(0.75));
+  g->area = GTK_DRAWING_AREA(dtgtk_drawing_area_new_with_aspect_ratio(9.0 / 16.0));
```

Comment:
Can we put this `9.0 / 16.0` constant somewhere and use it in all places ? We can probably declare such constant into iop/iop_api.h ? And name it IOP_GRAPH_ASPECT_RATIO ?

Reply:
done, also for the modules already having 16:9 ratio for their graph are